<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogTRSESettings</class>
 <widget class="QDialog" name="DialogTRSESettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>807</width>
    <height>606</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Turbo Rascal Syntax Error Settings</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QTabWidget" name="tabWidget">
       <property name="currentIndex">
        <number>2</number>
       </property>
       <widget class="QWidget" name="tab">
        <attribute name="title">
         <string>General</string>
        </attribute>
        <widget class="QWidget" name="gridLayoutWidget_4">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>50</y>
           <width>491</width>
           <height>121</height>
          </rect>
         </property>
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="1" column="0">
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>Memory Analyzer Font Size</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="leFontSizeMemoryAnalyzer">
            <property name="inputMask">
             <string>99</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
       <widget class="QWidget" name="tab_2">
        <attribute name="title">
         <string>Environment</string>
        </attribute>
        <widget class="QWidget" name="gridLayoutWidget">
         <property name="geometry">
          <rect>
           <x>70</x>
           <y>40</y>
           <width>521</width>
           <height>152</height>
          </rect>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="2">
           <widget class="QPushButton" name="btnEmulator">
            <property name="text">
             <string>...</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QPushButton" name="btnDasm">
            <property name="text">
             <string>...</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Dasm location</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="leDasm"/>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="leEmulator"/>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Emulator location</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Exomizer location</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="leExomizer"/>
          </item>
          <item row="2" column="2">
           <widget class="QPushButton" name="btnExomizer">
            <property name="text">
             <string>...</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
       <widget class="QWidget" name="Compiler">
        <attribute name="title">
         <string>Compiler</string>
        </attribute>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>30</y>
           <width>551</width>
           <height>221</height>
          </rect>
         </property>
         <property name="title">
          <string>Zeropages</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_9">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <property name="sizeConstraint">
             <enum>QLayout::SetMinAndMaxSize</enum>
            </property>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_10">
              <item>
               <widget class="QPushButton" name="pushButton_2">
                <property name="text">
                 <string>Zeropage Help</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_3">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
               <widget class="QLabel" name="label_7">
                <property name="text">
                 <string>Zeropages:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="leZeropages"/>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="1,2,2">
              <item>
               <widget class="QLabel" name="label_11">
                <property name="text">
                 <string>Decrunch zero pages</string>
                </property>
               </widget>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_7">
                <item>
                 <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0">
                  <item>
                   <layout class="QHBoxLayout" name="horizontalLayout">
                    <item>
                     <widget class="QLineEdit" name="leDecrunchZp1"/>
                    </item>
                    <item>
                     <widget class="QLineEdit" name="leDecrunchZp2"/>
                    </item>
                    <item>
                     <widget class="QLineEdit" name="leDecrunchZp3"/>
                    </item>
                    <item>
                     <widget class="QLineEdit" name="leDecrunchZp4"/>
                    </item>
                   </layout>
                  </item>
                 </layout>
                </item>
               </layout>
              </item>
              <item>
               <widget class="QLabel" name="label_13">
                <property name="font">
                 <font>
                  <pointsize>6</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>*note: The 2nd zeropage here also uses the following zp+1 zeropage address (so using $48 implies using both $48 and $49)</string>
                </property>
                <property name="scaledContents">
                 <bool>false</bool>
                </property>
                <property name="wordWrap">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_8">
              <item>
               <widget class="QLabel" name="label_12">
                <property name="text">
                 <string>Screenmemory zero page</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="leZeropageScreenMemory"/>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_13" stretch="1,2,2">
              <item>
               <widget class="QLabel" name="label_14">
                <property name="text">
                 <string>Workhorse zeropages</string>
                </property>
               </widget>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_14">
                <item>
                 <layout class="QVBoxLayout" name="verticalLayout_5" stretch="0">
                  <item>
                   <layout class="QHBoxLayout" name="horizontalLayout_15">
                    <item>
                     <widget class="QLineEdit" name="leInternalZp1"/>
                    </item>
                    <item>
                     <widget class="QLineEdit" name="leInternalZp2"/>
                    </item>
                    <item>
                     <widget class="QLineEdit" name="leInternalZp3"/>
                    </item>
                    <item>
                     <widget class="QLineEdit" name="leInternalZp4"/>
                    </item>
                   </layout>
                  </item>
                 </layout>
                </item>
               </layout>
              </item>
              <item>
               <widget class="QLabel" name="label_15">
                <property name="font">
                 <font>
                  <pointsize>6</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>These are used internally in TRSE for increased speed</string>
                </property>
                <property name="scaledContents">
                 <bool>false</bool>
                </property>
                <property name="wordWrap">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QGroupBox" name="groupBox_3">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>270</y>
           <width>211</width>
           <height>91</height>
          </rect>
         </property>
         <property name="title">
          <string>Exomizer</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0">
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="1,2">
                <item>
                 <widget class="QLabel" name="label_10">
                  <property name="text">
                   <string>Hide exomizer footprint: </string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QCheckBox" name="chkExomizerFootprint">
                  <property name="text">
                   <string/>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
       <widget class="QWidget" name="tab_3">
        <attribute name="title">
         <string>Editor</string>
        </attribute>
        <widget class="QComboBox" name="cmbTheme">
         <property name="geometry">
          <rect>
           <x>160</x>
           <y>20</y>
           <width>241</width>
           <height>22</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label_3">
         <property name="geometry">
          <rect>
           <x>40</x>
           <y>20</y>
           <width>55</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>Theme</string>
         </property>
        </widget>
        <widget class="QGroupBox" name="groupBox">
         <property name="geometry">
          <rect>
           <x>40</x>
           <y>100</y>
           <width>461</width>
           <height>311</height>
          </rect>
         </property>
         <property name="title">
          <string>Editor behavior</string>
         </property>
         <widget class="QWidget" name="gridLayoutWidget_2">
          <property name="geometry">
           <rect>
            <x>70</x>
            <y>50</y>
            <width>301</width>
            <height>121</height>
           </rect>
          </property>
          <layout class="QGridLayout" name="gridLayout_4">
           <item row="2" column="1">
            <widget class="QLineEdit" name="leFontSize">
             <property name="inputMask">
              <string>99</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="leTabWidth">
             <property name="inputMask">
              <string>99</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Tab width</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>Font size</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
        <widget class="QComboBox" name="cmbPalette">
         <property name="geometry">
          <rect>
           <x>160</x>
           <y>50</y>
           <width>241</width>
           <height>22</height>
          </rect>
         </property>
         <item>
          <property name="text">
           <string>Dark</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Light</string>
          </property>
         </item>
        </widget>
        <widget class="QLabel" name="label_8">
         <property name="geometry">
          <rect>
           <x>40</x>
           <y>50</y>
           <width>101</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>Window palette</string>
         </property>
        </widget>
       </widget>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="text">
          <string>OK</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
